openedx/core/djangoapps/credit/migrations/0008_auto__del_index_creditprovider_provider_id__chg_field_crediteligibilit.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Changing field 'CreditEligibility.provider'
+        db.alter_column('credit_crediteligibility', 'provider_id', self.gf('django.db.models.fields.related.ForeignKey')(null=True, to=orm['credit.CreditProvider']))
+
+    def backwards(self, orm):
+        # Changing field 'CreditEligibility.provider'
+        db.alter_column('credit_crediteligibility', 'provider_id', self.gf('django.db.models.fields.related.ForeignKey')(default=1, to=orm['credit.CreditProvider']))
```

Comment:
I see so the default will be 1 for any new ones that are created after this goes live?  In that case can we use a value that would indicate a provider that is special in some way.  I'm worried about associating all rows with now provider ID getting reset to a provider that does not make sense.  Given that this feature has never been turned on in production, I think you are probably ok to do what you are doing, but is there an impact to the open-source community for a change like this?